fire_atlas_editor/lib/services/storage/web.dart

```diff
@@ -0,0 +1,107 @@
+import 'dart:async';
+import 'dart:html';
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:fire_atlas_editor/services/storage/storage.dart';
+import 'package:fire_atlas_editor/store/store.dart';
+import 'package:flame_fire_atlas/flame_fire_atlas.dart';
+
+class FireAtlasStorage extends FireAtlasStorageApi {
+  static final Storage _localStorage = window.localStorage;
+
+  Future<LoadedProjectEntry> loadProject(String path) {
+    final value = _localStorage[path];
+    if (value == null) {
+      throw 'Unknow project with id $path';
+    }
+
+    final entry = LoadedProjectEntry(
+      path,
+      _readBase64Project(value),
+    );
+
+    return Future.value(entry);
```

Comment:
if you make the func async you don't need to wrap right? not sure what I prefer though, both are good

Reply:
Good point, changed